### PR TITLE
feat:Service version interface configuration

### PR DESCRIPTION
**Pull request by jinronga**
**Description**

This PR feature #

**Notes for Reviewers**
Please add the following content to .env.backup and env:
BACKEND_SERVER_VERSION=v0.1.8
TITLE="Kirin API"
TIMEZONE="UTC"
MILVUS_VERSION=v2.3.12

**[Signed commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by jinronga** [comment]
this issues  174

File: backend/src/api/routes/version.py [hunk 1]
```diff
@@ -26,4 +27,8 @@
     status_code=fastapi.status.HTTP_200_OK,
 )
 async def get_version() -> dict:
-    return {"version": settings.VERSION}
\ No newline at end of file
+    return ServiceVersionResponse(
```
Comment (Aisuko) [inline backend/src/api/routes/version.py:30]: cc @Micost  Which way we should use?

**Comment by Micost** [comment]
@jinronga please take a look of the comment in code review and check errors

**Comment by jinronga** [comment]
Sorry I didn't check the return type

**Review by Micost — APPROVED** [review]
LGTM

**Pull request merged**